lambdas/search/index.py

```diff
@@ -78,34 +78,44 @@ def lambda_handler(request):
         _source = []
     elif action == 'images':
         body = {
-            'query': {
-                'terms': {
-                    "ext": IMG_EXTS
-                }
-            }
+            'query': {'terms': {'ext': IMG_EXTS}},
+            'sort': [{'last_modified': {'order': 'desc'}}],
```

Comment:
Sorts are very expensive--e.g., the query will need to fetch all the files to check their timestamps instead of stopping after finding enough matches. Let's not sort.

Reply:
moved to inner_hits